### PR TITLE
[Intel MKL] Adding support for DNNL1 builds to public CI.

### PR DESCRIPTION
**Pull request by claynerobison**
@penpornk This is also not required for r2.2

### CODE REVIEW

File: tensorflow/tools/ci_build/linux/mkl/build-dev-container.sh [hunk 1]
```diff
@@ -119,6 +121,11 @@ function build_container()
     TF_DOCKER_BUILD_ARGS+=("--build-arg ENABLE_SECURE_BUILD=--secure-build")
   fi
 
+  #Add build arg for DNNL1
```
Comment (penpornk) [inline tensorflow/tools/ci_build/linux/mkl/build-dev-container.sh]: Nit:
```suggestion
  # Add build arg for DNNL1
```

**Review by penpornk — CHANGES_REQUESTED** [review]
Thank you for the notes and for the PR! I have one minor formatting nit.

**Comment by claynerobison** [comment]
Got it. Thanks for the feedback, @penpornk!

**Review by penpornk — APPROVED** [review]
Thank you for the quick fix!

**Pull request merged**